pandas/core/dtypes/common.py

```diff
@@ -1859,7 +1859,7 @@ def _validate_date_like_dtype(dtype) -> None:
     try:
         typ = np.datetime_data(dtype)[0]
     except ValueError as e:
-        raise TypeError("{error}".format(error=e))
+        raise TypeError(f"{e}")
```

Comment:
hmm, I not keen on f-strings used like this where equivalent to just `str(e)`. However, in this case, I _think_ you could just pass the ValueError directly to the TypeError constructor.
```suggestion
        raise TypeError(e)
```

Reply:
Will do
Thanks for the comment!